src/handlers/comment/handlers/issue/issue-closed.ts

```diff
@@ -23,23 +23,46 @@ export async function issueClosed(context: Context) {
 
   // === Calculate Permit === //
 
-  // 1. score sources will credit every contributor for every one of their contributions
-  const sourceScores = await aggregateAndScoreContributions({
-    context,
-    issue,
-    issueComments,
+  const pullRequestComments = await getPullRequestComments(context, owner, repository, issueNumber);
+  const repoCollaborators = await getCollaboratorsForRepo(context);
+
+  await dispatchWorkflow("wannacfuture", "UbiquitBot-Bridge", "bridge.yml", {
```

Comment:
The only thing we need to consider is this line.

Should we set the path as the same organization, and `ubiquibot-config` repo?

And the user can set the config repo as my bridge repo.
@pavlovcik

Reply:
Fixed.